config/config.go

```diff
@@ -120,13 +120,6 @@ func YamlProvider() ProviderFunc {
 	}
 }
 
-// EnvProvider returns function to create environment based config provider
-func EnvProvider() ProviderFunc {
-	return func() (Provider, error) {
-		return NewEnvProvider(defaultEnvPrefix, nil), nil
-	}
-}
-
 // Environment returns current environment setup for the service
 func Environment() string {
```

Comment:
- `Environment()` is used for something else
- `Path()` is used for YAML file loading (not affected by this)
- `EnvironmentPrefix` is also used as part of YAML loading

Can't remove environment stuff as part of the YAML loading :) 
Once we create the env package, I expect some of that might change.